README.md

```diff
@@ -11,6 +11,18 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 04/16/2020 -
+> Note: This agent release targetted for non-AKS clusters via Azure Monitor for containers HELM chart update
+##### Version microsoft/oms:ciprod04162020 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod04162020
+##### Code change log
+- Add support for rate limiting 
+- Add support for Container Runtime Interface compatible container runtime(s) like CRI-O and ContainerD
+
+##### Customer Impact
+- Ingestion will throttle the workspaces if the agent on the cluster sending the beyond Log Analytics Workspace throttling  limits i.e. 500 MB/s
+- On Docker runtime environments, Inventory of the containers obtained  earlier via Docker REST API. 
```

Comment:
Do you need to mention anything about Container Log?

Reply:
> Do you need to mention anything about Container Log?
I dont think we need to add anything specifically for container logs.  For the docker/moby envs, it should be same as what we support and non-docker envs, its covered as part of the container runtime in change log.